scss/utilities/mixins/molecules/_footer-menus.scss

```diff
@@ -0,0 +1,37 @@
+@charset "UTF-8";
+
+//
+// Footer Menu Mixins
+//
+// Mixins for the menus that appear in the global footer.
+//
+// Style guide: Mixins.Molecules.Footer Menus
+//
+
+@mixin footer-menus {
```

Comment:
This mixin is not needed and does not provide any value vs writing the code once in a selector.